chapters/VK_QNX_screen_surface/platformCreateSurface_screen.txt

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 BlackBerry Limited.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[[platformCreateSurface_screen,platformCreateSurface_screen]]
+
+=== QNX Screen Platform
+
+[open,refpage='vkCreateScreenSurfaceQNX',desc='Create a slink:VkSurfaceKHR object for a QNX Screen window',type='protos']
+--
+
+To create a sname:VkSurfaceKHR object for a QNX Screen surface, call:
+
+include::{generated}/api/protos/vkCreateScreenSurfaceQNX.txt[]
+
+  * pname:instance is the instance to associate the surface with.
+  * pname:pCreateInfo is a pointer to a slink:VkScreenSurfaceCreateInfoQNX
+    structure containing parameters affecting the creation of the surface
+    object.
+  * pname:pAllocator is the allocator used for host memory allocated for the
+    surface object when there is no more specific allocator available (see
+    <<memory-allocation,Memory Allocation>>).
+  * pname:pSurface is a pointer to a slink:VkSurfaceKHR handle in which the
+    created surface object is returned.
+
+include::{generated}/validity/protos/vkCreateScreenSurfaceQNX.txt[]
+--
+
+[open,refpage='VkScreenSurfaceCreateInfoQNX',desc='Structure specifying parameters of a newly created QNX Screen surface object',type='structs']
+--
+
+The sname:VkScreenSurfaceCreateInfoQNX structure is defined as:
+
+include::{generated}/api/structs/VkScreenSurfaceCreateInfoQNX.txt[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * pname:flags is reserved for future use.
+  * pname:context and pname:window are QNX Screen code:context and code:window
+    to associate the surface with.
+
+.Valid Usage
+****
+  * [[VUID-VkScreenSurfaceCreateInfoQNX-display-00000]]
```

Comment:
@oddhack the VUs will need to be generated correctly

Reply:
Hi, is it something I can do myself or it should be fixed by Khronos only?

Reply:
Normally these are emitted so it would just be

```
  * my vu text
```

and there is a script Jon runs that will generate these, the script makes sure a new number is assigned globally

Reply:
Right. Please remove the placeholder VUID anchors, they get inserted by a script after the PR is accepted to main branch.

Reply:
Should it be [[]] or [[[[VUID-VkScreenSurfaceCreateInfoQNX-display]]] in the text? Sorry, I never used them before.

Reply:
The entire anchor is inserted by the script, so it should be as @sfricke-samsung mentioned above, with no placeholder anchor at all:

> \* my vu text

Reply:
I added the fix for this along with some minor text fixes